Utils/azure-explorer-common/src/com/microsoft/tooling/msservices/serviceexplorer/Node.java

```diff
@@ -389,4 +401,28 @@ public Node createNode(Node parent, String sid, NodeContent content) {
     public String getServiceName() {
         return TelemetryConstants.ACTION;
     }
-}
\ No newline at end of file
+
+    public void openResourcesInPortal(HasId resource, String subscriptionId) throws AzureCmdException {
+        try {
+            final AzureManager azureManager = AuthMethodManager.getInstance().getAzureManager();
+            // not signed in
+            if (azureManager == null) {
+                return;
+            }
+            final String portalUrl = azureManager.getPortalUrl();
+            final String tenantId = azureManager.getTenantIdBySubscription(subscriptionId);
+            String url = portalUrl
+                    + REST_SEGMENT_JOB_MANAGEMENT_TENANTID
+                    + tenantId
+                    + REST_SEGMENT_JOB_MANAGEMENT_RESOURCE
+                    + resource.id();
+            if (Desktop.isDesktopSupported() && Desktop.getDesktop().isSupported(Desktop.Action.BROWSE)) {
```

Comment:
shall we use BrowserUtil.browse provided by intellj and Program.launch in eclipse?

Reply:
Maybe we can call `DefaultLoader.getIdeHelper().openLinkInBrowser(linkUrl)`. There are different implementation of IDEHelper in IntelliJ and Eclipse

Reply:
Thanks for the advice, changed to use `IDEHelper`